bash/convert_dtm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
the convert_dtm.sh and reproject_dtm are meant to be temporary I assume until we can get the data-library pipeline working, might just want to think about writing documentation to address this and highlight why you might need to use these scripts

Reply:
You addressed this in the notes in the PR write up